src/generate-manifest.js

```diff
@@ -118,6 +119,20 @@ const processPrivateVars = function (generatedManifest, codiusVars) {
   addPrivateVarEncodings(generatedManifest)
 }
 
+const processContainers = function (generatedManifest) {
```

Comment:
this could be called `removeEmptyEnvironments` instead, because that's more descriptive of what it does